### DISCUSSION TITLE
In version History, system is coming as a user whenever version note is added

### DISCUSSION THREAD

**Discussion by Karthi96** [pimcore/pimcore#9664] [Q&A / Support] [answered]
Hi Team, 

I'm adding version notes dynamically on the dataobject but whenever I do this I'm getting system as user modification. Here is the code that I'm using to set the version notes. 

```
 $object->save([ObjConst::VERSION_NOTE => $versionNote]);
```
Is there any way where we can set the explicit user to the object or how do we fix it? I'm using the 6.8.4 version 
![Screenshot from 2021-07-03 11-25-07](https://user-images.githubusercontent.com/17947667/124344784-25f03580-dbf2-11eb-89af-16f864ede67f.png)

**Comment by Karthi96** [comment, marked as answer]
When I set an explicit user modification Id, it works. 

```
$object->setUserModification(ListenerUtils::getUserId());
```

**Reply by dvesh3** [reply]
or better just call `Tool\Admin::getCurrentUser()->getId()`
